lib/pages/new_group/new_group.dart

```diff
@@ -27,11 +32,15 @@ class NewGroupController extends State<NewGroup> with ComparablePresentationCont
   final fetchContactsController = FetchContactsController();
   final contactStreamController = StreamController<Either<Failure, GetContactsSuccess>>();
   final groupNameTextEditingController = TextEditingController();
+  final createRoomStreamController = CreateRoomController();
 
   final selectedContactsMapNotifier = ValueNotifier<Map<PresentationContact, bool>>({});
   final haveSelectedContactsNotifier = ValueNotifier(false);
   final haveGroupNameNotifier = ValueNotifier(false);
   final isEnableEEEncryptionNotifier = ValueNotifier(true);
+  final avatarNotifier = ValueNotifier<MatrixFile?>(null);
+
+  final int maxFileSizeDefault = 5 * 1024 * 1024;
```

Comment:
you should clarify this variable. is it in MB or KB or GB ?. `maxFileSizeDefault` => `maxFileSizeDefaultInMB`

Reply:
You are right!